docs/serving/samples/hello-world/helloworld-go/Dockerfile

```diff
@@ -1,24 +1,25 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.12 as builder
+FROM golang:1.13 as builder
 
 # Copy local code to the container image.
-WORKDIR /go/src/github.com/knative/docs/helloworld
-COPY . .
+WORKDIR /app
+COPY . ./
```

Comment:
I wonder if this should do the `go mod download` trick, since many people will copy this? I know it's been decided on in the past, but consider adding it.

(I'm not attached to adding it - definitely makes things more complex, but it might be worth it.)

Reply:
Added go mod download and modified go build to use -mod=readonly for strictness of the build.